app/views/admin/stats.html.erb

```diff
@@ -0,0 +1,51 @@
+<h1>Admin Stats</h1>
+
+<%
+
+# See: http://stackoverflow.com/a/1341318/1323144
+def mean(array)
+  array.inject{ |sum, el| sum + el }.to_f / array.size
+end
+
+# See: http://stackoverflow.com/a/14859546/1323144
+def median(array)
```

Comment:
Hopefully a temporarily solution; would be nice to do it within the database instead.